test/processIssueComment.test.js

```diff
@@ -35,6 +36,31 @@ describe('Process Issue Comment', () => {
         )
         expect(mockContext.github).not.toHaveBeenCalled()
     })
+
+    // test('If there is nil contributions, inform about it', async () => {
```

Comment:
This test looks good to me!

Reply:
The only thing is that the part of `parseIssueComment` being tested doesn't work properly if there's no actual issue/comment being linked to it and I'm unsure if whether the function is missing some data we need for the mock testing or if there's a way to simulate a real issue where the comment is being posted in.

Reply:
I think its fine to add another 'export' from this file to make testing easier (so we can test `processAddContributor` directly.

Reply:
We have a few of the end-to-end tests in here: https://github.com/all-contributors/all-contributors-bot/blob/master/test/tasks/processIssueComment/probot-app-e2e.test.js. Generally we try to reserve end-to-end tests for the core use cases. You could add one here too tho.

Reply:
I've had a look at that test file and the issue thread it uses but I don't know how I could go about making sure it looks at the right comment (as it seems to look at the whole thread).